### ISSUE TITLE
SRM net

### ISSUE THREAD

**Issue by EvgenyKashin** [implus/PytorchInsight#3]
Hi, cool repository! Can you consider adding to comparison SRM net from "SRM : A Style-based Recalibration Module for Convolutional Neural Networks" (https://arxiv.org/abs/1903.10829) which looks very promising. You can use SRM block implementation from this repository https://github.com/EvgenyKashin/SRMnet

**Comment by implus** [comment]
Thanks for suggestions! We will work on it later.

**Comment by hyunjaelee410** [comment]
Hi, thank you for sharing your work!

When training SRMNet on ImageNet, I found it useful not to apply weight decay on SRM parameters. Since SRM module learns to how to utilize mean and std statistics, it is better not to impose weight decay regularization.

You can refer to 
https://github.com/hyunjaelee410/style-based-recalibration-module/blob/master/imagenet.py#L327
https://github.com/hyunjaelee410/style-based-recalibration-module/blob/master/models/recalibration_modules.py#L17-L19

and @EvgenyKashin  thanks for your interest in our work and sharing your implementation!

**Comment by EvgenyKashin** [comment]
@hyunjaelee410 it is very cool insight about weight decay. Thank you for sharing official implementation of your work!

**Comment by shiyuanyin** [comment]
@implus 
Hi, great repository!  
I'd like to ask some questions.
Have you tried other backbone networks as a comparison?
I want to know  its universality  and apply to other network?
for example, the SEnet can be used in many scenarios and networks
